src/Elements/Bundle/AlternateObjectTreesBundle/LevelDefinition/Input.php

```diff
@@ -52,15 +52,19 @@ public function getGroupedValues($condition, $offset = null, $limit = null)
         // create condition
         $where = $this->condition == '' ? 1 : $this->condition;
         if ($condition) {
+            $where = 'LIKE \'%'.$where.'%\'';
             $where .= ' AND ' . $condition;
         }
+        // TODO: this is such a bonkers way to pass down values.
+        // When ViCA is built, we have to do this clean and simple.
```

Comment:
What is ViCA? If we are going to do this for sure would not be better just to do it than having a TODO comment?

Reply:
ViCA is going to be a completely different project in a new repo.
this TODO is to remind me that this case has to be handled in a better way

Reply:
I mean unless someone comes to here when start building VICA and then notice that they need to do same as this but in a clean way I do not see value of this TODO

Reply:
yeah.. i removed to comment to avoid confusion